### PR TITLE
chore: fix the links in project readme

### LINKED ISSUE #1017: [Broken Links] The  "contributing to ORAS" and  "development guide" in the readme are  broken links
### What happened in your environment?

This is the link I am talking about
iss

It takes you to https://oras.land/docs/contributing . It should rather take you to https://oras.land/docs/community/contributing_guide . Same is the case with [development guide](https://oras.land/docs/CLI/developer_guide)
iss2

### What did you expect to happen?

_No response_

### How can we reproduce it?

-

### What is the version of your ORAS CLI?

-

### What is your OS environment?

-

### Are you willing to submit PRs to fix it?

- [X] Yes, I am willing to fix it.

### PR DESCRIPTION
**Pull request by amands98**
This PR fixes the links. Fixes #1017 

**Please check the following list**:
- [ ]  Does the affected code have corresponding tests, e.g. unit test, E2E test?
- [ ]  Does this change require a documentation update?
- [ ]  Does this introduce breaking changes that would require an announcement or bumping the major version?
- [ ]  Do all new files have an appropriate license header?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,11 @@
 ## Docs
 
 Documentation for the ORAS CLI is located on
-the project website: [oras.land/cli](https://oras.land/docs/category/cli)
+the project website: [oras.land/cli](https://oras.land/docs/category/oras-commands)
 
 ## Development Environment Setup
 
-Refer to the [development guide](https://oras.land/docs/CLI/developer_guide) to get started [contributing to ORAS](https://oras.land/docs/contributing).
+Refer to the [development guide](https://oras.land/docs/how_to_guides/developer_guide) to get started [contributing to ORAS](https://oras.land/docs/community/contributing_guide).
```
Comment (TerryHowe) [inline README.md:18]:
```suggestion
Refer to the [development guide](https://oras.land/docs/community/developer_guide) to get started [contributing to ORAS](https://oras.land/docs/community/contributing_guide).
```

**Review by TerryHowe — CHANGES_REQUESTED** [review]

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Comment by qweeah** [comment]
LGTM but @amands98 can you help rebase the PR to main branch, thanks.

**Comment by amands98** [comment]
First PR merged, more to come.

**Pull request merged**